Makefile

```diff
@@ -1,5 +1,6 @@
 # OASIS_START
 # DO NOT EDIT (digest: a3c674b4239234cbbe53afe090018954)
```

Comment:
You might want to remove the `# OASIS_START` and `# OASIS_STOP` markers as the next invocation of `oasis setup` will overwrite your changes.